.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,27 @@
+**Android version and Phone Model**
+
+Add the Android version and the phone model on which the issue was encountered.
```

Comment:
Please separate this between a bug, chore feature request

Like

Is it a ...

- [ ] Feature Request
- [ ] Bug Report
- [ ] Chore

Then add this section in last for bug request and add two sections for feature request and chore

Thanks :smile: